packages/x-charts/src/models/seriesType/line.ts

```diff
@@ -66,6 +66,11 @@ export interface LineSeriesType
    * @default 'monotoneX'
    */
   curve?: CurveType;
+  /**
+   * If `true`, step curve starts and end at the first and last point.
+   * By default the line is extended to fill the space before and after.
+   */
+  strictStepCurve?: boolean;
```

Comment:
Would addind a `curve: 'strict-step'` option a cleaner approach than this flag? 🤔

Reply:
Not really because we also have `step-before` and `step-after`.

I do not expect people to use this flag. The docs is more here to highlight the special behavior on band scale than to encourage using it.